actions/websocket_actions.test.jsx

```diff
@@ -149,6 +150,23 @@ describe('handlePostEditEvent', () => {
     });
 });
 
+describe('handlePostUnreadEvent', () => {
+    test('post marked as unred', async () => {
+        const msgData = {last_viewed_at: 123, msg_count: 40, mention_count: 1};
+        const expectedData = {lastViewedAt: 123, msgCount: 40, mentionCount: 1, channelId: 'channel1'};
+        const expectedAction = {type: 'POST_UNREAD_SUCCESS', data: expectedData};
+        const msg = {
+            data: msgData,
+            broadcast: {
+                channel_id: 'channel1',
+            },
+        };
+
+        handlePostUnreadEvent(msg);
+        expect(store.dispatch).toHaveBeenCalledWith(expectedAction);
```

Comment:
And what should I expect in the reducer after this was dispatched?

Reply:
These tests have mocked out the reducers and store state, so we'd have to rewrite them to be able to test that. The handling for `POST_UNREAD_SUCCESS` actions are already tested in redux though, so I don't think we need any additional tests here